markdownreveal/commands.py

```diff
@@ -54,15 +54,22 @@ def cli():
     default=8123,
     help='Listen on port (default: 8123).',
 )
+@click.option(
+  '-n',
+  '--no-warmup',
+  is_flag=True,
+  help='Do not display the warmup slide, even if it exists in the'
+       ' style folder (default: false).'
+)
 def show(markdown_file: Path, host: str = 'localhost', port: int = 8123):
```

Comment:
It seems while merging, you lost the `no_warmup` parameter in `show()`. :stuck_out_tongue_winking_eye: